.github/workflows/kubeapps-general.yaml

```diff
@@ -34,12 +34,11 @@ env:
   KUBEAPPS_REPO: "vmware-tanzu/kubeapps"
   BRANCH_KUBEAPPS_REPO: "main"
   README_GENERATOR_REPO: "bitnami-labs/readme-generator-for-helm"
-  DOCKER_VERSION: "20.10.18"
```

Comment:
This var seems to be no longer used, I removed the mention in the release-process.md too @beni0888 , just double check that, and the other changes I made to release-process.md are OK in your eyes :)